docs/SETUP.md

```diff
@@ -68,7 +68,7 @@ cp $project_name/settingslocal.py.example $project_name/settingslocal.py
 # load the demo configuration
 ./manage.py loaddata dumps/example.json
 
-# create a super user
+# create a super user NOTE: it work for loggin in to the admin, to use it for open-connect login "attributes" column in "auth_permission" table must be filled in
```

Comment:
mmhh... the admin is something that belong to staff, and it's necessary to login in the admin backend interface.

the user.attributes instead belonge to the identity fields, and it's required to login through OIDC.

So I'd explain this in an accurate way